frontend/test/metabase/lib/formatting/filesize-integration.unit.spec.js

```diff
@@ -0,0 +1,161 @@
+// Integration tests for file size formatting feature
```

Comment:
These are not integration tests, they test `formatNumber` in isolation. The tests for some reason create some unnecessary data structures, then call formatNumber on some properties of it and perform assertions on these properties which practically is not different from tests in `frontend/src/metabase/lib/formatting/numbers.unit.spec.ts`

Reply:
Ok, so you happy for me to drop these set of tests completely?